tests/functional/reset_password.js

```diff
@@ -620,14 +625,21 @@ define([
         .findByCssSelector('form input.email')
           .getAttribute('value')
           .then(function (resultText) {
-            // email address should be pre-filled from the query param.
-            assert.equal(resultText, email);
+            // email address should not be pre-filled from the query param.
+            assert.equal(resultText, '');
           })
         .end()
 
         // ensure there is no back button when browsing directly to page
         .then(Test.noElementById(self, 'fxa-tos-back'))
 
+        // fill in email
+        .findByCssSelector('form input.email')
```

Comment:
You can use the [type](https://github.com/mozilla/fxa-content-server/blob/f1337b228ae027bdc988165f2b86b1487457e2da/tests/functional/lib/helpers.js#L919) helper here.